src/platform/packages/shared/kbn-unified-tabs/src/components/tab/tab.tsx

```diff
@@ -114,7 +133,7 @@ export const Tab: React.FC<TabProps> = (props) => {
               title={item.label}
```

Comment:
```suggestion
```

Reply:
I must have lost this change during merge conflicts, thanks!